example/quantization/imagenet_gen_qsym_mkldnn.py

```diff
@@ -208,13 +208,15 @@ def save_params(fname, arg_params, aux_params, logger=None):
     if args.model == 'imagenet1k-resnet-152':
         rgb_mean = '0,0,0'
         rgb_std = '1,1,1'
-        excluded_sym_names += ['flatten0']
+        # TODO, fc1 will be enabled after MKL-DNN 0.18
+        excluded_sym_names += ['flatten0', 'fc1']
         if exclude_first_conv:
             excluded_sym_names += ['conv0']
     elif args.model == 'imagenet1k-inception-bn':
         rgb_mean = '123.68,116.779,103.939'
         rgb_std = '1,1,1'
-        excluded_sym_names += ['flatten']
+        # TODO, fc1 will be enabled after MKL-DNN 0.18
```

Comment:
v0.18 will be available soon.  @TaoLv is WIP for the upgrading.
Could we wait several days for v0.18?

Reply:
For now, these three examples will fail due to limitation of quantized FullyConnected (as it's enabled in current script). 
If v0.18 will be merged into master in a short time, then this would be fine.